app/client/src/pages/organization/CreateOrganizationForm.tsx

```diff
@@ -19,18 +19,20 @@ export const CreateApplicationForm = (
     onCancel: () => void;
   },
 ) => {
-  const { error, handleSubmit, pristine, submitting } = props;
+  const { error, handleSubmit, pristine, submitting, invalid } = props;
+  const onlySpaces = value => (!value.trim().length)
   return (
     <Form onSubmit={handleSubmit(createOrganizationSubmitHandler)}>
       {error && !pristine && <FormMessage intent="danger" message={error} />}
       <FormGroup intent={error ? "danger" : "none"} helperText={error}>
-        <TextField name="name" placeholder="Name" />
+        <TextField name="name" placeholder="Name"  validate={onlySpaces}/>
       </FormGroup>
       <FormFooter
         onCancel={props.onCancel}
         divider
         data-cy="t--create-org-submit"
         submitOnEnter
+        disabled={invalid}
```

Comment:
There is no prop called `disabled` defined in `FormFooter` hence TypeScript will break the build, you can make use of `canSubmit` prop instead.
```suggestion
        canSubmit={!invalid}
```